godtools/App/Flows/ToolScreenShare/ToolScreenShareFlow.swift

```diff
@@ -12,7 +12,9 @@ import Combine
 
 class ToolScreenShareFlow: Flow {
 
-    private let toolSettingsObserver: ToolSettingsObserver
+    typealias ToolScreenShareSettingsObserver = ToolSettingsObserver & RemoteShareable
```

Comment:
Oh whoa I didn't know that would work, basically placing both types into a single type.